src/utils/openwhisk.js

```diff
@@ -131,6 +132,17 @@ async function runPipeline(cont, pipe, actionParams) {
     const payload = {
       request: extractClientRequest(action),
     };
+    // if request has JSON body.content, pass it to payload.content
+    if (actionParams.body && action.request.headers['content-type'] === 'application/json') {
+      try {
+        const body = JSON.parse(actionParams.body);
```

Comment:
you don't need to parse it, as openwhisk already provides the `json` body as action param.